src/PKSim.Presentation/Presenters/Simulations/GlobalPKAnalysisPresenter.cs

```diff
@@ -78,6 +84,11 @@ public bool ShouldCalculateDDIRatio(string compoundName, string parameterName)
                 shouldCalculateGlobalPKParameter(compoundName, parameterName, CoreConstants.PKAnalysis.C_maxRatio);
       }
 
+      public bool ShouldCalculateAll()
+      {
+         return !GlobalPKAnalysis.Children.SelectMany(x => (x as IContainer).Children).Any();
```

Comment:
This x as IContaienr looks bad. No idea why this is done like this but this should be rewritten.
And please add comment explaining what your are testing here. 
Also this could be moved to GlobalPKAnalysis object and you create a property here. Tell don't ask